assignments/callbacks.js

```diff
@@ -24,25 +24,50 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
 
+getLength(items, function(length) {
```

Comment:
```suggestion
getLength(items, function(arr) {
    return arr.length
})
```
You don't need to change but just remember that the callback function can be the one to do the logic also, good concept to understand